### PR TITLE
Added Harvester for Ghent University 

### PR DESCRIPTION
**Pull request by ZobairAlijan**

### CODE REVIEW

**Comment by fabianvf** [comment]
The favicon should be named `ghent_favicon.ico`, not `ghent.ico`

**Comment by ZobairAlijan** [comment]
ops fixed it.

File: scrapi/harvesters/ghent_univ.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+"""
+
+from __future__ import unicode_literals
+
+from scrapi.base import OAIHarvester
+
+
+class GhentUniversity(OAIHarvester):
+    short_name = 'ghent'
+    long_name = 'ghent university'
```
Comment (fabianvf) [inline scrapi/harvesters/ghent_univ.py]: Could you capitalize this?

Comment (ZobairAlijan) [inline scrapi/harvesters/ghent_univ.py]: Absolutely

**Comment by erinspace** [comment]
@ZobairAlijan can you also remove the old img/favicons/ghent.ico file?

**Pull request merged**